dev/diffs/4.0.0.diff

```diff
@@ -2150,7 +2517,17 @@ index 795e9f46a8d..1797084a941 100644
        }
      }
    }
-@@ -1991,7 +2004,8 @@ abstract class ParquetFilterSuite extends QueryTest with ParquetTest with Shared
+@@ -1750,7 +1762,8 @@ abstract class ParquetFilterSuite extends QueryTest with ParquetTest with Shared
+     }
+   }
+ 
+-  test("SPARK-17091: Convert IN predicate to Parquet filter push-down") {
++  test("SPARK-17091: Convert IN predicate to Parquet filter push-down",
++      IgnoreComet("IN predicate is not yet supported in Comet, see issue #36")) {
```

Comment:
For other Spark version, looks like we are using IgnoreCometNativeScan("Comet has different push-down behavior")